---
layout: post
title: PRJ002MM Extraction protocol and matrices test run
date: 2021-10-25
categories: Aiptasia
tags: [sampling, 16s, microbiome manipulation, prj002-Aiptasia-MM]
---

**Objective:** 
The objective of today's experiment is twofold. 1. Test and confirm the extraction protocol modifications made using the Zymo kit. 2. compare lysis matrices in the bead beater to see which one returns a more optimal 

## Broader Context
Establishing relevance for how this fits in with larger research goals

## Protocol 
1. Using a prepared lysis tube (lysing matrix E without the bead from MP biomedicals) add 250 of FSW and your anemone sample to the tube
2. Bead beat for 1 minute at 8 m/s in for two rounds with a five minute pause between rounds.
3. Sample 50 ul of homogenate for use in CFU counts.
4. Add 500 ul of zymo shield.
5. Samples can now be frozen for later extraction or processed directly.
6. To extract: if needed defrost the samples 
7. Centrifuge the samples at 20C for 1 minute at 10,000xg.
8. Transfer the full volume of the sample into the III-F filter column in a collection tube.
9. Centrifuge the column and collection tube at 8,000xg for 1 minute
10. Discard the column keeping the eluent in the elution tube
11. Add 1200 ul of DNA binding buffer and pipette up and down to mix
12. In a second collection tube containing the II-CR filter. Add 800 ul of your sample mixture onto the column
13. Centrifuge the column at 10,000xg for 1 minute. Discard the flow through 
14. Repeat steps 12-13 with the remaining volume in the collection tube
15. Add 400ul of wash buffer 1 onto the column and centrifuge for 1 minute at 10,000xg. Discard the flow through
16. Add 700ul of wash buffer 2 onto the column and centrifuge for 1 minute at 10,000xg. Discard the flow through
17. Add 200ul of wash buffer 2 onto the column and centrifuge for 1 minute at 10,000xg. Discard the flow through
18. Transfer the column to a sterile 1.5ml microcentrifuge tube
19. Add 50 ul of nucleotide free water onto the column and let it incubate for 1 minute before centrifuging at 10,000xg for 1 minute
20. Repeat step 19 again adding another 50 ul. This will bring your final elution volume to 100ul. The II-CR filter can now be discarded.
21. Place the III-HRC filter in a new collection tube and add 600ul of HRC prep solution. The filter may appear like a crumbly powder. 
22. Centrifuge the III-HRC filter at 8,00xg for 3 minutes
23. Transfer the III-HRC column to a new sterile microcentrifuge tube. 
24. Transfer the eluted 100 ul of DNA from step 20 onto the III-HRC filter and centrifuge for 3 minutes at 16,000Xg. Discard the III-HRC filter, your samples are now ready for downstream processing.


## Raw Data
Sample | Conc (ng/ul) | 260/280 | 260/230 |
-------|--------------|---------|---------|
TS1    | 19.0         | 2.04    | 0.98    |
TS2    | 14.8         | 2.01    | 1.23    |
TS3    | 28.4         | 1.94    | 0.81    |
TS4    | 30.6         | 1.91    | 1.33    |
TS5    | 29.4         | 1.89    | 1.37    |
TS6    | 49.2         | 1.88    | 1.12    |  
TS7    | 1.6          | 2.55    | 0.26    |     

## Concluding Thoughts
Use of lysing matrix E will increase the quality and quantity of our  DNA for this extraction protocol; Aiptasia related experiments going forward should be done using this matrix. 
Overall the protocol should be sufficient for 16S microbial knockdown checks.